qiskit/providers/ibmq/managed/ibmqjobmanager.py

```diff
@@ -87,20 +97,21 @@ def run(
         Args:
             experiments: Circuit(s) or pulse schedule(s) to execute.
             backend: Backend to execute the experiments on.
-            name: Name for this set of jobs. Each job within the set will have
+            name: Name for this set of jobs.
+                Each job within the set will have
                 a job name that consists of the set name followed by a suffix.
-                Default: current datetime.
+                If not specified, the current date and time is used.
             max_experiments_per_job: Maximum number of experiments to run in each job.
                 If not specified, the default is to use the maximum allowed by
                 the backend.
                 If the specified value is greater the maximum allowed by the
                 backend, the default is used.
-            job_share_level: Allows sharing the jobs at the hub/group/project and
-                global level. The possible job share levels are: "global", "hub",
-                "group", "project", and "none". Default: "none".
-            job_tags: tags to be assigned to the job. The tags can
-                subsequently be used as a filter in the ``jobs()`` function call.
-                Default: None.
+            job_share_level: Allow sharing the jobs at the hub, group, project, or
+                global level. The level can be one of: ``global``, ``hub``,
+                ``group``, ``project``, and ``none``.
+            job_tags: Tags to be assigned to the jobs. The tags can
+                subsequently be used as a filter in the :meth:`IBMQBackend.jobs()`
```

Comment:
This wasn't cross referenced when I ran tox locally, should it contain the full path (i.e. :meth:`IBMQBackend.jobs()<qiskit.providers.ibmq.IBMQBackend.jobs>`)?